Remotion/Validation/UnitTests/Implementation/FallbackValidationMessageFactoryTest.cs

```diff
@@ -15,23 +15,84 @@
 // along with re-motion; if not, see http://www.gnu.org/licenses.
 // 
 using System;
+using Moq;
 using NUnit.Framework;
+using Remotion.Reflection;
+using Remotion.Validation.Implementation;
+using Remotion.Validation.UnitTests.TestDomain;
+using Remotion.Validation.Validators;
 
 namespace Remotion.Validation.UnitTests.Implementation
 {
   [TestFixture]
   public class FallbackValidationMessageFactoryTest
   {
     [Test]
-    [Ignore("RM-5906")]
     public void CreateValidationMessageForPropertyValidator ()
     {
+      var propertyValidatorMock = new Mock<IPropertyValidator>();
+      var validatedPropertyMock = new Mock<IPropertyInformation>();
+
+      var fallbackFactory = new FallbackValidationMessageFactory();
+      var result = fallbackFactory.CreateValidationMessageForPropertyValidator(propertyValidatorMock.Object, validatedPropertyMock.Object);
+
+      Assert.That(result, Is.Not.Null);
+      Assert.IsInstanceOf<InvariantValidationMessage>(result);
+      Assert.That(result.ToString(), Is.EqualTo($"{propertyValidatorMock.Object.GetType().Name}: Validation error."));
```

Comment:
I think we typically want to write the full expected thing. This does not tell me what output the toString method returns without having to jump through some thinking hoops.

Reply:
Since `validationMessage` is private, I have to use `.ToString()` to access the string. I am not sure how it can be done in a different way.

Reply:
I meant that the expected value should not use String.Format (which happens if you use `$"{someVariable}"`.

The inside of the assert should probably look like this:
`result.ToString(), Is.EqualTo("This is the actual string output without any formatting happening")`